charms/jupyter-ui/src/spawner_ui_config.yaml

```diff
@@ -211,11 +178,5 @@ spawnerFormDefaults:
     # value:
     #   - add-gcp-secret
     #   - default-editor
-    value:
-    # Added from https://github.com/kubeflow/kubeflow/pull/6160 to fix
-    # https://github.com/canonical/bundle-kubeflow/issues/423.  This was not yet in
-    # upstream and if they go with something different we should consider syncing with
-    # upstream.
-    # Auto-selects "Allow access to Kubeflow Pipelines" button in Notebook spawner UI
-    - access-ml-pipeline
```

Comment:
This is a fix we've applied, not something that comes from upstream.  Is this addressed automatically now by upstream?

Reply:
Seems like it is not addressed yet. Will put add it back.